data/json/items/armor/torso_clothes.json

```diff
@@ -1799,12 +1799,12 @@
       },
       {
         "id": "cthulhuprez1",
-        "name": { "str": "Cthulhu for President t-shirt" },
+        "name": { "str": "Cthulhu for President t-shirt", "str_pl": "Cthulhu for President t-shirts" },
         "description": "A t-shirt with the text 'Cthulhu for President: Why vote for the lesser evil'."
       },
       {
         "id": "cthulhuprez2",
-        "name": { "str": "Cthulhu for President t-shirt" },
+        "name": { "str": "Cthulhu for President t-shirt", "str_pl": "Cthulhu for President t-shirts" },
```

Comment:
...why? The ``s`` is added at the end automatically if no plural form is defined

Reply:
because basic build test failed with "can't make plural form, use str_pl please thank you"